OSVR-Unity/Assets/OSVRUnity/Resources/OsvrDistortion.shader

```diff
@@ -18,66 +18,59 @@
 /// limitations under the License.
 /// </copyright>
 /// <summary>
-/// Based on Unity's (Pro Only) Image Effect: Fisheye.shader
+/// Long ago, based on Unity's (Pro Only) Image Effect: Fisheye.shader
```

Comment:
Looked at the fisheye shader to get a general idea of how image effects worked. The vertex shader is still the same, but many simple vertex shader examples use that same code or something very similar. The fragment shader is where the magic happens, which differs significantly. For reference, here is the fisheye shader: https://github.com/leapmotion-examples/unity/blob/master/v1/leapdj/Assets/Standard%20Assets/Image%20Effects%20(Pro%20Only)/_Sources/Shaders/FisheyeShader.shader